packages/graphql-server/src/functions/graphql.ts

```diff
@@ -31,16 +36,17 @@ import { renderPlaygroundPage } from 'graphql-playground-html'
 import { BaseLogger } from 'pino'
 import { v4 as uuidv4 } from 'uuid'
 
-import { ApolloError, AuthContextPayload } from '@redwoodjs/api'
-import { getAuthenticationContext } from '@redwoodjs/api'
+import { CorsConfig, createCorsContext } from 'src/cors'
```

Comment:
Would it be ok to use relative imports here, don't think it'll be too many `..`?

Reply:
Done.